### ISSUE TITLE
Support for Opening Multiple Links in External Browser

### ISSUE THREAD

**Issue by touwys** [yanus171/Handy-News-Reader#858]
At the moment, it is possible to open a link from Handy, in various ways, to the default external browser.

Please consider to add support for _selecting multiple articles in a feed list_, by long-pressing on each, to subsequently _open their links in the default external browser, simultaneously._

Thank you.

**Comment by yanus171** [comment]
Explain me again why do you need it


touwys ***@***.***> 11 марта 2023 г. 11:54:26 написал:

**Comment by touwys** [comment]
> Explain me again why do you need it touwys

Being able to open multiple articles in the external browser simultaneously,  not only saves a lot of time over having to open them one by one, but, importantly, the external browser affords an easier way to share, and to distribute numerous articles over different devices, via synchronisation. For instance: a PC, a mobile phone and a mobile tablet.

___

**Comment by yanus171** [comment]
Possible you should write to https://github.com/yanus171/Handy-News-Reader/issues/125

**Comment by touwys** [comment]
> Possible you should write to #125

No, this request does not fit there. This is a straight forward request to have Handy open multiple links in an external browser — nothing more.

___

**Comment by yanus171** [comment]
I suppose this is impossible by web browser app
